### DISCUSSION TITLE
How can I trigger Inference Jobs to run again after changing a language model?

### DISCUSSION THREAD

**Discussion by tehniemer** [karakeep-app/karakeep#436] [Q&A]
All of my Inference Jobs have failed, and I want to try again with a new model, but I can't seem to figure out how to trigger them to run again.

**Comment by MohamedBassem** [comment]
There's currently no way to trigger inference on its own (sounds like a reasonable feature request). If you want to trigger it now, you'll have to trigger a recrawl as well (admin dashboard > recrawl all links)

**Comment by kamtschatka** [comment]
There is now an option in the admin panel called "Regenerate AI Tags for All Bookmarks", which you can click to trigger the inference again.
